src/locales/index.js

```diff
@@ -18,20 +18,70 @@ const mobileTranslations = {
   es: esMobile
 }
 
-const unsupportedLocaleTag = () => {
+const defaultLanguage = () => {
+  const polyglot = new Polyglot({locale: defaultLocaleTag, phrases: availableTransations[defaultLocaleTag] , onMissingKey: () => null})
+  if (!isDesktop) polyglot.extend(mobileTranslations[defaultLocaleTag])
+  return polyglot
+}
+
+
+const extendPolyglot = (locale, polyglot, phrases, mobilePhrases) => {
+  polyglot.locale(locale)
+  polyglot.extend(phrases)
+  if (!isDesktop) { polyglot.extend(mobilePhrases) }
+  return polyglot
+}
+
+const findMissingKeys = (defaultKeys, customKeys) => {
+  const newTransaltionsSet = new Set(customKeys)
+  const missingKeys =  defaultKeys.filter(element => !newTransaltionsSet.has(element))
+  if (missingKeys.length) { console.warn('Missing keys:', missingKeys) }
+}
+
+const flattenKeys = (phrases, prefix = '') => {
```

Comment:
I thought polyglot did this for free?

Reply:
It does, but if I call the `extend` function over the previous polyglot instance, I cannot detect if there are any missing keys, unless I create a new instance of polyglot just to flatten the keys for comparison purposes and then extend the default instance with the new strings. Having multiple instances of the polyglot object might be a bit confusing, that's why I've decided to implement a function just to flatten the keys. Although, this might be a problem if polyglot is updated and they change the way they build the phrases object. What's your take on this @rfreitas?

Reply:
Made it into a one line 😬